packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -262,8 +266,9 @@ class SliderThemeData extends Diagnosticable {
       disabledThumbColor: primaryColorDark.withAlpha(disabledThumbAlpha),
       overlayColor: primaryColor.withAlpha(overlayLightAlpha),
       valueIndicatorColor: primaryColor.withAlpha(valueIndicatorAlpha),
-      thumbShape: const RoundSliderThumbShape(),
-      valueIndicatorShape: const PaddleSliderValueIndicatorShape(),
+      thumbShape: new RoundSliderThumbShape(),
```

Comment:
I think these two parameters should still be `const Foo()`.

Reply:
Oh, right, I changed them because that was one side of effect of my earlier design.  Reverted.